custom/nutrition_project/ucr/reports/growth_monitoring_v1.json

```diff
@@ -0,0 +1,131 @@
+{
+  "domains": [
+    "india-nutrition-project"
+  ],
+  "server_environment": [
+    "india"
+  ],
+  "report_id": "static-growth_monitoring_v1",
+  "data_source_table": "static-weight_height_v1",
+  "config": {
+    "title": "Growth Monitoring V1 (Static)",
+    "description": "Last value of indicators filled per child care case per month",
+    "visible": true,
+    "aggregation_columns": [
+      "child_care_case_id",
+      "gender",
+      "supervisor_id",
+      "flw_id",
+      "month"
+    ],
+    "filters": [
+      {
+        "display": "Filter by FLW",
+        "slug": "flw_id",
+        "type": "dynamic_choice_list",
+        "field": "userID",
+        "choice_provider": {
+          "type": "location"
+        },
+        "ancestor_expression": {
+          "field": "supervisor_id",
+          "location_type": "supervisor"
+        },
+        "datatype": "string"
+      },
+      {
+        "display": "Filter by Supervisor",
+        "slug": "supervisor_id",
+        "type": "dynamic_choice_list",
+        "field": "supervisor_id",
+        "choice_provider": {
+          "type": "location"
+        },
+        "datatype": "string"
+      },
+      {
+        "display": "Form Completed On",
+        "slug": "completed_time",
+        "type": "date",
+        "field": "completed_time",
+        "datatype": "date"
+      },
+      {
+        "display": "Date of Birth",
+        "slug": "dob",
+        "type": "date",
+        "field": "dob",
+        "datatype": "date"
+      },
+      {
+        "display": "Gender",
+        "slug": "gender",
+        "type": "choice_list",
+        "field": "gender",
+        "datatype": "string",
+        "choices": [
+          {
+            "display": "Male",
+            "value": "M"
+          },
+          {
+            "display": "Female",
```

Comment:
Do you think more adding more indexes with these columns would be beneficial here seeing there are more filters in this report.

Reply:
thanks for flagging that @Rohit25negi . I cross checked that gender and completed time are not being used in the app, so i have removed them. The only one is dob which is used to check for age, so the checks are like dob - today = certain number of days, I guess an index won't help there.

Reply:
offlined. Looks good